arbitrum-docs/node-running/reference/arbos-software-releases/arbos20.mdx

```diff
@@ -5,9 +5,12 @@ sidebar_position: 2
 author: dlee
 ---
 
-ArbOS 20 Atlas is shipped via Nitro v2.3.0, which is available on Docker hub with the image tag: `offchainlabs/nitro-node:v2.3.0-3e14543`. This release of Nitro is a mandatory upgrade for Arbitrum One and Nova validators.For Arbitrum One and Nova, the ArbOS 20 upgrade requires a governance vote to activate.
+ArbOS 20 Atlas is shipped via Nitro v2.3.1, which is available on Docker hub with the image tag: `offchainlabs/nitro-node:v2.3.1-26fad6f`. This release of Nitro is a mandatory upgrade for Arbitrum One and Nova validators.For Arbitrum One and Nova, the ArbOS 20 upgrade requires a governance vote to activate.
```

Comment:
If v2.3.0 is the first version to support ArbOS 20, shouldn't we mention that version here?
Maybe changing the text to something like "supported since Nitro v2.3.0 ..."

Reply:
Hey good catch, 2.3.0 is indeed the minimum version but we made some critical fixes in 2.3.1 that are important for validators so I decided to just use 2.3.1.